### ISSUE TITLE
People that finished the course, still showing in a specific project.

### ISSUE THREAD

**Issue by BrunoBMFT** [codam-coding-college/find-peers#14]
Small bug, on the website, that are people that finished the common core, and they are still register on the project on your website.

![image](https://github.com/codam-coding-college/find-peers/assets/61157658/bb04ed22-0df1-4437-8329-aead97d4ceef)

![image](https://github.com/codam-coding-college/find-peers/assets/61157658/3bce84c2-d4f8-4aef-a946-8d58e5197d11)

the person that I'm reporting the bug is rporcopim and anfreire, one is finishing the course (rprocopi) and anfreire finished a while ago

**Comment by GroteGnoom** [comment]
Yes, that's the information intra gives us. Those users are subscribed to the project, even though that's not relevant anymore. I don't think we have the data that's necessary to know who's finished the core. I do think we could see who has finished cub3d, I think those are the people we want to filter out. And then we should do it for all projects where finishing one project excludes the other.
